lib/cfnguardian/resources/dms_task.rb

```diff
@@ -0,0 +1,42 @@
+module CfnGuardian::Resource
+  class DMSTask < Base
+    
+    def default_alarms             
+      alarm = CfnGuardian::Models::DMSTaskAlarm.new(@resource)
+      alarm.name = 'CDCLatencySourceCritical'
+      alarm.metric_name = 'CDCLatencySource'
+      alarm.statistic = 'Minimum'
+      alarm.threshold = 30
+      alarm.evaluation_periods = 10
+      alarm.enabled = false
+      @alarms.push(alarm)
+      
+      alarm = CfnGuardian::Models::DMSTaskAlarm.new(@resource)
```

Comment:
over-ride required for alarm action to warning?

Reply:
Yeah we don't want to monitor latency unless it's for a DMS task that is expected to run 24/7 (confirmed with Jared when we were talking about it, if this is enabled by default, tasks that are set to run on a schedule / infrequently will be constantly alarming)